Makefile

```diff
@@ -194,7 +194,7 @@ deploy: # Deploy/redeploy previously built REANA cluster.
 	if [ $$(docker images | grep -c '<none>') -gt 0 ]; then \
 		docker images | grep '<none>' | awk '{print $$3;}' | xargs docker rmi; \
 	fi && \
-	helm install ${TRUNC_INSTANCE_NAME} helm/reana $(addprefix --set , ${CLUSTER_FLAGS}) $(addprefix -f , ${VALUES_YAML_PATH}) --wait && \
+	helm install ${TRUNC_INSTANCE_NAME} helm/reana $(addprefix --set , ${CLUSTER_FLAGS}) $(addprefix -f , ${VALUES_YAML_PATH}) --wait --namespace ${INSTANCE_NAME} --create-namespace && \
```

Comment:
Just for the record. `--create-namespace` was only introduced in [Helm 3.2.0 release](https://github.com/helm/helm/releases/tag/v3.2.0) last April. We should check if we specify/recommend a minimum version somewhere, otherwise, this will fail (`Error: unknown flag: --create-namespace`).

I see that, for instance, we download a lower version in Travis to do`make test`, although it's just to run `helm lint` so it shouldn't affect.

Reply:
Thanks for the heads up :)
- [Addressed in docs](https://github.com/reanahub/docs.reana.io/pull/47/files#diff-1752afb578474f66f308d74ae48e9e06R7) 
- [Addressed in Travis](https://github.com/reanahub/reana/pull/337/files#diff-354f30a63fb0907d4ad57269548329e3R31)